include/flatbuffers/grpc.h

```diff
@@ -104,7 +104,8 @@ class SliceAllocator : public Allocator {
   }
 
   virtual uint8_t *reallocate_downward(uint8_t *old_p, size_t old_size,
-                                       size_t new_size) override {
+                                       size_t new_size, size_t in_use_back,
+                                       size_t in_use_front) override {
```

Comment:
Thanks for adding these parameters.. they have to be used as well, though, the `memcpy` below is now invalid, it has to do the same as the two `memcpy` statements in `Allocator::reallocate_downward`. Better yet, pull those 2 mempy's into a method and call it from here.

Reply:
Done